python/paddle/utils/dlpack.py

```diff
@@ -28,7 +28,7 @@ def to_dlpack(x):
     Encodes a tensor to DLPack.
 
     Args:
-        x (Tensor): A tensor, and the data type is bool, float32, float64, int32, int64.
```

Comment:
why delete it?  you can add more type

Reply:
Since almost all data types can be supported, no need to emphasize.

Reply:
Since almost all data 
types('bool'，'float16'，'float32'，'float64'，'int8'，'int16'，'int32'，'int64'，'uint8'，'complex64'，'complex128') can be supported, no need to emphasize here.